documentation/en/configuration/default-curio-configuration.md

```diff
@@ -552,4 +552,79 @@ description: The default curio configuration
     # type: string
     #WebHookURL = ""
 
+
+[HTTP]
+  # DomainName specifies the domain name that the server uses to serve HTTP requests.
+  #
+  # type: string
+  #DomainName = ""
+
+  # CertCacheDir path to the cache directory for storing SSL certificates needed for HTTPS.
+  #
+  # type: string
+  #CertCacheDir = ""
```

Comment:
I worry that letsencrypt won't be particularly happy when one tries to run the cluster with multiple http nodes, might be trying to put this in the DB somehow